lib/ThreeScaleClient.php

```diff
@@ -94,7 +107,7 @@ public function getHost() {
    */
   
      public function authorize($appId, $appKey = null, $credentials_or_service_id, $usage = null) {
-    $url = "http://" . $this->getHost() . "/transactions/authorize.xml";
+    $url = "http://" . $this->getHost() . ":" . $this->getPort() . "/transactions/authorize.xml";
```

Comment:
Make sense.

Also, should we use`https` or keep it as it is `http`. I saw your comments on other plugins on persistent https connections and was wondering what should we do here.